README.md

```diff
@@ -64,8 +64,13 @@ we test it on 200 patients dataset, got the **AUC** as below:
 |AVG_MAX|No|
 |AVG_MAX_LSE|No|
 
-> We are now implementing heatmap options for `AVG_MAX`, `AVG_MAX_LSE`, and `PROB`, So before that you can use the supported `global_pool` options to train a model and plot the heatmaps.
+* We also provide heatmap comparision here, including AVG, [LSE](https://arxiv.org/abs/1705.02315), and our own PROB pooling.
 
+||&emsp;&emsp;&emsp;&emsp; original &emsp;&emsp;&emsp;&emsp; [AVG (dev mAUC:0.895)](config/example_AVG.json) &emsp; [LSE (dev mAUC:0.896)](config/example_LSE.json) &emsp; [PROB (dev mAUC:0.896)](config/example_PROB.json)|
+|------|-----|
+|Cardiomegaly|<img src="Cardi.jpg" width="100%" align="middle"/>|
```

Comment:
The heatmap looks so weird here, like the Atelectasis and Consolidation, why is it like bounding box? It feels some bbox annotation is mixed with the training. @yww211 @deadpoppy

Reply:
> The heatmap looks so weird here, like the Atelectasis and Consolidation, why is it like bounding box? It feels some bbox annotation is mixed with the training. @yww211 @deadpoppy

That is just a coincidence, more examples do not have this pattern.